src/components/form/form-summary/form-summary.style.js

```diff
@@ -0,0 +1,65 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+
+const StyledFormSummary = styled.div`
+  display: inline-flex;
+  align-items: center;
+  font-size: 13px;
+  font-weight: 700;
+  margin: -8px;
+  white-space: nowrap;
+  padding: 8px;
+  border-radius: 4px;
+ 
+  ${({ isInvalid, theme }) => isInvalid && css`
+    background-color: ${theme.form.invalid};
+    margin-left: 15px;
+  `}
+`;
+
+export const StyledInternalSummary = styled.span`
+  padding: 0 3px;
+
+  carbon-form-summary__icon {
```

Comment:
Since this is being used as a CSS class in `src/components/form/form-summary/form-summary.component.js`:
```jsx
<Icon className='carbon-form-summary__icon' type={ key } />
```
shouldn't its name start with a `.` here? (`.carbon-form-summary__icon`)

But shouldn't we be using StyledComponents instead of class names?

Reply:
Eagle eyes, great spot!

Reply:
:white_check_mark: